package.json

```diff
@@ -13,7 +13,7 @@
     "generate-api-docs": "node ./scripts/generate-api-docs.js"
   },
   "dependencies": {
-    "@ampproject/filesize": "^2.1.1",
+    "@ampproject/filesize": "^4.3.0",
```

Comment:
Despite bumping 2 major versions, this works exactly the same for our purposes. No changes needed elsewhere!